test/k6/src/tests/instances.js

```diff
@@ -319,13 +371,17 @@ export default function (data) {
       TC09_CompleteConfirmInstance(data);
       TC10_SoftDeleteInstance(data);
       TC11_HardDeleteInstance(data);
+      TC12_GetInstances_ByPersonNumber(data);
+      TC13_GetInstances_ByOrgNumber(data);
     } else {
       // Limited test set for use case tests
       var instanceId = TC01_PostInstance(data);
       data.instanceId = instanceId;
       TC02_GetInstanceById(data);
       TC03_GetInstances_PartyFilter(data);
       TC11_HardDeleteInstance(data);
+      TC12_GetInstances_ByPersonNumber(data);
```

Comment:
ref our convo on Slack, Khan. These lines would break the use case tests in all environments the code has not been deployed to yet. I would recommend to have the k6 tests in a seperate PR, so we can test successfully before merging the automated tests.  Or in the least removing these two lines, so it only effects the regression tests